datafusion/physical-plan/src/projection.rs

```diff
@@ -446,13 +578,30 @@ impl ProjectionStream {
 }
 
 /// Projection iterator
-struct ProjectionStream {
+pub struct ProjectionStream {
```

Comment:
```suggestion
struct ProjectionStream {
```

Let's make this change in another PR